mne/io/utils.py

```diff
@@ -0,0 +1,164 @@
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Matti Hamalainen <msh@nmr.mgh.harvard.edu>
+#          Martin Luessi <mluessi@nmr.mgh.harvard.edu>
+#          Denis Engemann <denis.engemann@gmail.com>
+#          Teon Brooks <teon.brooks@gmail.com>
+#          Marijn van Vliet <w.m.vanvliet@gmail.com>
+#          Mainak Jas <mainak.jas@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+
+def _find_channels(ch_names, ch_type='EOG'):
+    """Helper to find EOG channel.
+    """
+    substrings = (ch_type)
```

Comment:
I suspect this is broken

(ch_type) -> [ch_type] or (ch_type,)

also docstring should mention that it can find more than EOG